resources/lang/en/admin/categories/general.php

```diff
@@ -17,7 +17,6 @@
     'no_default_eula'					=> 'No primary default EULA found. Add one in Settings.',
     'update'  							=> 'Update Category',
     'use_default_eula'					=> 'Use the <a href="#" data-toggle="modal" data-target="#eulaModal">primary default EULA</a> instead.',
-    'use_default_eula_column'			=> 'Use default EULA',
```

Comment:
This deletion seems weird, but maybe the message has been subsumed into the following? (`use_default_eula_disabled`).